clang-tools-extra/clang-reorder-fields/ReorderFieldsAction.cpp

```diff
@@ -116,26 +118,78 @@ findMembersUsedInInitExpr(const CXXCtorInitializer *Initializer,
   return Results;
 }
 
-/// Returns the full source range for the field declaration up to (not
-/// including) the trailing semicolumn, including potential macro invocations,
-/// e.g. `int a GUARDED_BY(mu);`.
+/// Returns the next token after `Loc` (including comment tokens).
+static std::optional<Token> getTokenAfter(SourceLocation Loc,
```

Comment:
[follow-up idea] it looks like in clang-tidy there is a somewhat similar helper function findNextTokenIncludingComments 
(in LexerUtils.h). Perhaps, if they could be unified /  moved to a common place - this would be wonderful.

Reply:
Thanks for the pointer, I looked at the code in clang-format as I expected to find something like this there, but I did not think of clang-tidy.  I'll send a followup to move the code to `Lexer.h`.